software-engineer-in-test-mid-level.md

```diff
@@ -0,0 +1,72 @@
+# Software Engineer in Test - Mid Level
```

Comment:
remove the mid-level also from the file name - if we don't say senior it should be enough